app/routes/account.$accountId.settings.members/components/InviteMemberForm/InviteMemberFrom.tsx

```diff
@@ -42,6 +46,7 @@ const InviteMemberFrom = () => {
                 type="email"
                 w="300px"
               />
+              <input hidden readOnly name="account_name" value={accountName as string} />
```

Comment:
is this so we can use it in the email template?

Reply:
Yes, we pass it to to the email template and notification if it exists